public/stylesheets/vertical.css

```diff
@@ -0,0 +1,34 @@
+@media (min-width: 768px) {
```

Comment:
Do we need one more styles file ? 
Application loading slows down with more files.